src/Composer/Installers/FeatherBBInstaller.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace Composer\Installers;
+
+class FeatherBBInstaller extends BaseInstaller
+{
+    protected $locations = array(
```

Comment:
Indeed we changed a lot of code and structure since I submitted this PR. New locations are up to date in the latest commit !